src/openforms/emails/templates/emails/templatetags/appointment_information.txt

```diff
@@ -20,4 +23,4 @@
 
 {% trans "If you want to cancel or change your appointment, you can do so below." %}
 {% trans "Cancel appointment" %}: {{ appointment_cancel_link }}
-{% trans "Change appointment" %}: {{ appointment_change_link }}{% endautoescape %}
+{% trans "Change appointment" %}: {{ appointment_change_link }}{% endif %}{% endautoescape %}
```

Comment:
Does Django template language support naming the endif? This one points all the way up to the start.

Reply:
afaik it doesn't